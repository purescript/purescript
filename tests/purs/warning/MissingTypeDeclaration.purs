-- @shouldWarnWith MissingTypeDeclaration
module Main where

x = 0
