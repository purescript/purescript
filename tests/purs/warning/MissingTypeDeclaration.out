Warning found:
in module [33mMain[0m
at tests/purs/warning/MissingTypeDeclaration.purs:4:1 - 4:6 (line 4, column 1 - line 4, column 6)

  No type declaration was provided for the top-level declaration of [33mx[0m.
  It is good practice to provide type declarations as a form of documentation.
  The inferred type of [33mx[0m was:
  [33m     [0m
  [33m  Int[0m
  [33m     [0m

in value declaration [33mx[0m

See https://github.com/purescript/documentation/blob/master/errors/MissingTypeDeclaration.md for more information,
or to contribute content related to this warning.

